app/dao/fact_notification_status_dao.py

```diff
@@ -156,7 +156,7 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
     query = db.session.query(
         *([
             Template.name.label("template_name"),
-            Template.is_precompiled_letter,
+            False,  # TODO: this is related to is_precompiled_letter
```

Comment:
Yeah, the issue isn't the model, it's that I'm not confident enough with what happens downstream from here to figure out what needs to change. From now on, this will always be false, so it should be safe. Maybe it's an obvious thing, but it's eluding me right now.